modin/pandas/io.py

```diff
@@ -706,7 +738,7 @@ def read_xml(
     )
 
 
-@_inherit_docstrings(pandas.HDFStore, apilink="pandas.HDFStore")
+@_inherit_docstrings(pandas.HDFStore)
```

Comment:
Why do we add links to pandas docs in other places, for example, for read_spss? If there is no need for this, let's remove as part of a separate PR if it doesn't fail the job.

Reply:
In other places link is added to show the user information that is common for pandas and Modin. Currently `pd.read_spss` looks as follow:
```
    Load an SPSS file from the file path, returning a DataFrame.

    <Supported API data should placed here in the future>

    Notes
    -----
    See pandas API documentation for `pandas.read_spss <https://pandas.pydata.org/pandas-docs/version/1.4.2/reference/api/pandas.read_spss.html>`_ for more.
```

For `HDFStore` and `ExcelFile` there is no need to add links to pandas since all these fucntions are inherited and contain pandas docstrings.

Reply:
ok